x-pack/plugins/watcher/public/sections/watch_list/components/watch_list.tsx

```diff
@@ -0,0 +1,267 @@
+                  id: 'xpack.watcher.sections.watchList.watchTable.menuEditButtonDescription',
+                  defaultMessage: 'Edit watch',
+                })}
+                onClick={() => {
+                  urlService.change(
+                    `/management/elasticsearch/watcher/watches/watch/${watch.id}/edit`
+                  );
+                }}
+              >
+                <FormattedMessage
+                  id="xpack.watcher.sections.watchList.watchTable.menuEditButtonTitle"
+                  defaultMessage="Edit"
+                />
+              </EuiButtonEmpty>
+            );
+          },
+        },
+      ],
+    },
+  ];
+  const selectionConfig = {
+    onSelectionChange: setSelection,
+  };
+  const pagination = {
+    initialPageSize: 10,
+    pageSizeOptions: [10, 50, 100],
+  };
+  const searchConfig = {
+    box: {
+      incremental: true,
+    },
+    toolsRight: (
+      <EuiButton
+        data-test-subj="btnDeleteWatches"
+        onClick={() => {
+          setWatchesToDelete(selection.map(selected => selected.id));
+        }}
+        color="danger"
+        disabled={!selection.length}
+      >
+        <FormattedMessage
+          id="xpack.watcher.sections.watchList.deleteWatchButtonLabel"
+          defaultMessage="Delete"
+        />
+      </EuiButton>
+    ),
+  };
+  return (
+    <EuiPageContent>
+      <DeleteWatchesModal
+        callback={(deleted?: string[]) => {
+          if (deleted) {
+            setWatches(
+              watches.filter(watch => {
+                return !deleted.includes(watch.id);
+              })
+            );
+          }
+          setWatchesToDelete([]);
+        }}
+        watchesToDelete={watchesToDelete}
+      />
+      <EuiFlexGroup justifyContent="spaceBetween" alignItems="flexEnd">
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="m">
+            <h1>
+              <FormattedMessage
+                id="xpack.watcher.sections.watchList.header"
+                defaultMessage="Create threshold alert"
+              />
+            </h1>
+          </EuiTitle>
+          <EuiSpacer size="s" />
+          <EuiText size="s" color="subdued">
+            <p>
+              <FormattedMessage
+                id="xpack.watcher.sections.watchList.subhead"
+                defaultMessage="Send out emails, slack messages and log events when specific parameters are hit"
+              />
+            </p>
+          </EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer />
+      <EuiButton
+        data-test-subj="createThresholdAlertButton"
+        onClick={() => {
+          urlService.change('/management/elasticsearch/watcher/watches/new-watch/threshold');
+        }}
+      >
+        <FormattedMessage
+          id="xpack.watcher.sections.watchList.createThresholdAlertButtonLabel"
+          defaultMessage="Create threshold alert"
+        />
+      </EuiButton>{' '}
```

Comment:
Can we wrap these buttons in a flex group instead of putting spaces between them? This will help the layout become responsive on narrower screens.

```jsx
<EuiFlexGroup gutterSize='m'>
  <EuiFlexItem grow={false}>
    <EuiButton
      data-test-subj="createThresholdAlertButton"
      onClick={() => {
        urlService.change('/management/elasticsearch/watcher/watches/new-watch/threshold');
      }}
    >
      <FormattedMessage
        id="xpack.watcher.sections.watchList.createThresholdAlertButtonLabel"
        defaultMessage="Create threshold alert"
      />
    </EuiButton>
  </EuiFlexItem>

  <EuiFlexItem grow={false}>
    <EuiButton
      data-test-subj="createAdvancedWatchButton"
      onClick={() => {
        urlService.change('/management/elasticsearch/watcher/watches/new-watch/json');
      }}
    >
      <FormattedMessage
        id="xpack.watcher.sections.watchList.createAdvancedWatchButtonLabel"
        defaultMessage="Create advanced watch"
      />
    </EuiButton>
  </EuiFlexItem>
</EuiFlexGroup>
```

Wide screen:

![image](https://user-images.githubusercontent.com/1238659/53759954-626ec380-3e76-11e9-922d-3f487aaebc06.png)

Narrow screen:

![image](https://user-images.githubusercontent.com/1238659/53759959-639ff080-3e76-11e9-8e67-da7aa3d083bf.png)

Reply:
done